airflow/scripts/discord_scripts/discord_bot.py

```diff
@@ -0,0 +1,100 @@
+import asyncio
+from datetime import datetime, timedelta
```

Comment:
Order it by import types and alphabetic:
<local python imports>

<third party>

<your imports>

example:
import asyncio
from datetime import datetime, timedelta
import os

import discord
from dotenv import load_dotenv